packages/dashboard/src/components/pages/home/home.tsx

```diff
@@ -1,17 +1,193 @@
+                        <Text>
+                            Binary Bot
+                        </Text>
+                    ),
+                },
+            ],
+        },
+        {
+            default: false,
+            icon: 'IcWalletTradeTypes',
+            label: 'Trade Types',
+            subroutes: [
+                {
+                    label: 'CFDs',
+                    value: () => (
+                        <Text>
+                            CFDs
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Multipliers',
+                    value: () => (
+                        <Text>
+                            Multipliers
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Options',
+                    value: () => (
+                        <Text>
+                            Options
+                        </Text>
+                    ),
+                },
+            ],
+        },
+        {
+            default: false,
+            icon: 'IcWalletMarkets',
+            label: 'Markets',
+            subroutes: [
+                {
+                    label: 'Forex',
+                    value: () => (
+                        <Text>
+                            Forex
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Synthetic Indices',
+                    value: () => (
+                        <Text>
+                            Synthetic Indices
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Stock Indices',
+                    value: () => (
+                        <Text>
+                            Stock Indices
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Commodities',
+                    value: () => (
+                        <Text>
+                            Commodities
+                        </Text>
+                    ),
+                },
+            ],
+        },
+    ];
+
+    const list_groups:any[] = [];
+    const subroutes:any[] = [];
+    list.forEach(list_item => {
+        if (!list_item.subroutes){
+            subroutes.push(list_item);
+        } else {
+            list_item.subroutes.forEach(item => subroutes.push(item));
+        }
+    })
+
+    list.forEach(list_item => {
+        if (!list_item.subroutes){
+            list_groups.push(list_item);
+        } else {
+            list_groups.push({
+                icon: list_item.icon,
+                label: list_item.label,
+                subitems: (list_item.subroutes as any[]).map((subroute:any) => {
+                    return subroutes.findIndex((sub) => subroute === sub);
+                }),
+            });
+        }
+    })
```

Comment:
Since we're looping through the same list can we combine this into a single `forEach`?

Reply:
Ah right, I forgot to refactor this one